cmd/validator/validate_accounting.go

```diff
@@ -307,7 +320,9 @@ func resultsPrinter(config Params, printCurl bool, results <-chan Result) {
 
 			// Print error message if there is one.
 			if r.Error != nil {
+				numErrors++
 				errorLog.Printf("Processor error: %v\n", r.Error)
+				continue
 			}
 			// Print error details if there are any.
```

Comment:
It would be nice to move everything after this under an else statement. Not a fan of continue. :)